applications/style.css

```diff
@@ -46,6 +46,13 @@ body {
   text-align: center;
 }
 
+.footer {
+  width: 100vw;
+  text-align: center;
+  position: fixed;
+  bottom: 5px;
```

Comment:
Why here 5px and other place 100ox

Reply:
Fixed it as well.

Reply:
the numbers should be multiple of 4 as per design system

Reply:
Fixed it